### PR TITLE
 UI issues Resolved

### PR DESCRIPTION
**Pull request by kalash-qbatch**

### CODE REVIEW

File: src/components/PagesComponent/SoftwareDevelopers/style.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ const SoftwareDevelopersWrapper = styled.div`
           padding: 22px 48px;
         }
         p {
-          font-weight: 700;
+          font-weight: 700 !important;
```
Comment (ali-haider-qbatch) [inline src/components/PagesComponent/SoftwareDevelopers/style.js:91]: do not use important

Comment (kalash-qbatch) [reply src/components/PagesComponent/SoftwareDevelopers/style.js:91]: i am using 2 class at a same time and also both class nesting code are different thats why i have to add important if you have any suggestion then do let me know

Comment (ali-haider-qbatch) [reply src/components/PagesComponent/SoftwareDevelopers/style.js:91]: Give a className instead using direct html tag

Comment (kalash-qbatch) [reply src/components/PagesComponent/SoftwareDevelopers/style.js:91]: we are writing code in Sass not in CSS so are you this will not create any issue

**Pull request merged**